src/Graphics/OpenGLContext/opengl_ContextImpl.cpp

```diff
@@ -60,7 +62,7 @@ void ContextImpl::init()
 	}
 
 	{
-		if ((m_glInfo.isGLESX && (m_glInfo.bufferStorage && m_glInfo.majorVersion * 10 + m_glInfo.minorVersion > 32)) || !m_glInfo.isGLESX)
+		if ((m_glInfo.isGLESX && m_glInfo.bufferStorage) || !m_glInfo.isGLESX)
```

Comment:
Why was ```m_glInfo.majorVersion * 10 + m_glInfo.minorVersion > 32``` removed?

BufferedDrawer depends on glDrawElementsBaseVertex, which is only available in GLES 3.2+